src/inc/xclrdata.idl

```diff
@@ -2536,3 +2536,16 @@ interface IXCLRDataExceptionNotification4 : IXCLRDataExceptionNotification3
      */
     HRESULT ExceptionCatcherEnter([in] IXCLRDataMethodInstance* catchingMethod, DWORD catcherNativeOffset);
 }
+
+[
+    object,
+    local,
+    uuid(e77a39ea-3548-44d9-b171-8569ed1a9423)
+]
+interface IXCLRDataExceptionNotification5 : IXCLRDataExceptionNotification4
+{
+    /*
+     * New code was generated or discarded for a method.
```

Comment:
> or discarded

probably a copy paste error?

Reply:
I would also add a comment making it clear that nativeCodeLocation is the start address of the newly jitted code.